### DISCUSSION TITLE
Mocking Junit test case for RestClientBuilder

### DISCUSSION THREAD

**Discussion by sreekar12** [quarkusio/quarkus#41377] [Q&A]
Can you please help how to mock RestClientBuilder using @QuarkusComponentTest or any other Mocking functions like Mockito or powerMockito

Example : RestClientBuilder.newBuilder().baseUri(URI).build(HandlerService.class);

could you please provide a sample mocking example for above line.

**Comment by yrodiere** [comment]
Hey. I converted this from a GitHub issue. For usage questions, please use the Q&A category on GitHub discussions.

I don't personally have an answer, but hopefully someone else will.
